src/redux/type/NoteType.js

```diff
@@ -0,0 +1 @@
+export const ADDNOTE = "ADDNOTE";
```

Comment:
if letters are all UPPER CASE, we can add `_` as word separator 

```suggestion
export const ADD_NOTE = "ADD_NOTE";
```

code is now more readable